hyperspy/io_plugins/digital_micrograph.py

```diff
@@ -1001,59 +1025,82 @@ def get_mapping(self):
                     "Acquisition_instrument.TEM.Detector.EDS.real_time",
                     None),
             })
-        elif self.signal_type == "CL":
+        elif self.signal_type == "CL" or ("Acquisition" in image_tags_dict.keys() and "Monarc_Spectrometer" in image_tags_dict["Acquisition"].keys()):
```

Comment:
Include this in https://github.com/hyperspy/hyperspy/blob/d42c03d5986c649d2121fa34cfe361e9506e5a86/hyperspy/io_plugins/digital_micrograph.py#L627-L638?